src/Components/header.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import axios from 'axios';
+import { render } from '@testing-library/react';
```

Comment:
po co to? react ma funkcje render sam w sobie

Reply:
usunięte